packages/react/src/components/AccountSettings/ChangePassword/ChangePassword.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+
+import { Logger } from 'aws-amplify';
+import { changePassword, translate } from '@aws-amplify/ui';
+
+import { useAuth } from '../../../internal';
+import { View, Flex } from '../../../primitives';
+import {
+  DefaultCurrentPassword,
+  DefaultError,
+  DefaultNewPassword,
+  DefaultSubmitButton,
+} from './defaultComponents';
+import { ChangePasswordProps } from './types';
+import { FormValues } from '../types';
```

Comment:
Maybe we can start using `import type xxx`?

Reply:
I'm definitely open to the idea. I'm not too familiar with the syntax though, what's the benefit of using `import type` over `import`?

Reply:
As far as I know, it is an explicit way to tell TS compiler that it is safe to not emit a js file for that import since it is just importing typings.

Reply:
My two cents on this:
- we should be writing our _types.ts_ so that they do only contain types
- if we are going to start using the `import type` syntax it should be enforced by eslint (see rule [here](https://typescript-eslint.io/rules/consistent-type-imports/))

Reply:
This sounds like non-trivial (but important!) refactor in React source code. What do you think about:
- I'll enforce the convention of `types.ts` to contain only type files
- Let's do a refactor after account settings and file uploader components are merged to clean up types and use `import type` with eslint rules enforced.

Reply:
> I'll enforce the convention of types.ts to contain only type files for my upcoming PRs.

🎉

> Let's do a refactor after account settings and file uploader components are merged to clean up types and use import type with eslint rules enforced.

Cool! Good end of year item